fhir-tools/src/main/java/com/ibm/fhir/tools/CodeGenerator.java

```diff
@@ -1260,6 +1261,8 @@ private String getSyntaxBasedValueSetValidationMethod(String valueSet, String fi
         String suffix = fieldType + (isRepeating ? "s" : "");
         if (ALL_LANG_VALUE_SET_URL.equals(valueSet)) {
             return "checkLanguage" + suffix;
+        } else if (UCUM_UNITS_VALUE_SET_URL.equals(valueSet)) {
+            return "checkUcum" + suffix;
```

Comment:
I'm surprised it didn't hit Location as well.

Reply:
Not sure what you're referring to here - don't see any UCUM value sets used in the Location resource.